internal/ycsb/ycsb.go

```diff
@@ -85,3 +108,37 @@ func Run(ctx context.Context, dir string, args []string) (*config.TestResults, e
 
 	return res, nil
 }
+
+func parseMetrics(output string) map[string]Metrics {
+	res := make(map[string]Metrics)
+
+	scanner := bufio.NewScanner(strings.NewReader(output))
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		prefix := strings.Fields(line)[0]
+
+		switch prefix {
+		case "READ", "TOTAL", "UPDATE":
+			var m Metrics
+			var parsedPrefix string
+			_, err := fmt.Sscanf(line,
```

Comment:
@AlekSi, PTAL. Initially, I considered using regex, but a simple scan seems to be enough.

I will continue testing with more scenarios, but you can try this branch locally to see how it works. Please let me know if there's anything else apart from parsing and printing the parsed output that should be included in the task.

Thanks!